package.json

```diff
@@ -36,18 +30,17 @@
     "imports-loader": "^0.6.5",
     "iso-3166-2": "0.4.0",
     "isomorphic-fetch": "2.2.1",
-    "istanbul": "1.0.0-alpha.2",
     "jquery": "^2.2.0",
     "jsdom": "8.5.0",
     "jsdom-global": "1.7.0",
     "keycode": "2.1.1",
     "lodash": "4.11.2",
     "material-ui": "0.15.0",
-    "mocha": "2.4.5",
     "moment": "2.13.0",
     "node-sass": "3.7.0",
     "object.entries": "1.0.3",
     "ramda": "^0.22.1",
+    "raven-js": "^3.7.0",
```

Comment:
it looks like you added `coveralls` as well, that should be handled the same way.